src/Features/Core/Portable/Snippets/SnippetProviders/AbstractConsoleSnippetProvider.cs

```diff
@@ -59,22 +59,19 @@ protected override async Task<ImmutableArray<TextChange>> GenerateSnippetTextCha
 
         private async Task<TextChange> GenerateSnippetTextChangeAsync(Document document, int position, CancellationToken cancellationToken)
         {
-            var consoleSymbol = await GetSymbolFromMetaDataNameAsync(document, cancellationToken).ConfigureAwait(false);
-            Contract.ThrowIfNull(consoleSymbol);
             var generator = SyntaxGenerator.GetGenerator(document);
             var tree = await document.GetRequiredSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
             var token = tree.FindTokenOnLeftOfPosition(position, cancellationToken);
 
             // We know symbol is not null at this point since it was checked when determining
             // if we are in a valid location to insert the snippet.
-            var typeExpression = generator.TypeExpression(consoleSymbol);
             var declaration = GetAsyncSupportingDeclaration(token);
             var isAsync = declaration is not null && generator.GetModifiers(declaration).IsAsync;
 
             var invocation = isAsync
                 ? generator.AwaitExpression(generator.InvocationExpression(
-                    generator.MemberAccessExpression(generator.MemberAccessExpression(typeExpression, generator.IdentifierName(nameof(Console.Out))), generator.IdentifierName(nameof(Console.Out.WriteLineAsync)))))
-                : generator.InvocationExpression(generator.MemberAccessExpression(typeExpression, generator.IdentifierName(nameof(Console.WriteLine))));
+                    generator.MemberAccessExpression(generator.MemberAccessExpression(generator.IdentifierName(nameof(Console)), generator.IdentifierName(nameof(Console.Out))), "WriteLineAsync")))
+                : generator.InvocationExpression(generator.MemberAccessExpression(generator.IdentifierName(nameof(Console)), "WriteLine"));
```

Comment:
Why use a string literal instead of nameof? The result should be identical, right?